src/api/routes/users.ts

```diff
@@ -55,14 +55,41 @@ router.patch("/:userId", (req: Request, res: Response, next: NextFunction) => {
                 last_name = user.last_name,
                 email = user.email,
             } = jsonData;
-
-            res.status(200).json({ first_name: first_name, last_name: last_name, email: email });
+        res.status(200).json({ first_name: first_name, last_name: last_name, email: email });
```

Comment:
Add proper indentation.

Please refer to the documentation to install `pre-commit` hooks.
They will lint your files before committing.